webviz_dev_sync/webviz_dev_sync.py

```diff
@@ -4,7 +4,7 @@
 import os
 from pathlib import Path
 import shutil
-from pip._internal.utils.misc import get_distribution
+from pkg_resources import get_distribution
```

Comment:
In `webviz-config` we use `importlib.metadata` from std.lib, which is replacing A`setuptools`' `pkg_resources` afaik. https://github.com/equinor/webviz-config/blob/9ca2b6e9978c806c957c36cefb5db121afdc9a16/webviz_config/plugins/__init__.py#L5-L10